packages/db-objects-jsonl/lib/objects/objectsInMemServerRedis.js

```diff
@@ -505,6 +520,13 @@ class ObjectsInMemoryServer extends ObjectsInMemoryJsonlDB {
                     }
                     handler.sendBufBulk(responseId, Buffer.from(fileData));
                 }
+            } else if (namespace === this.metaNamespace) {
+                const result = this.getMeta(id);
+                if (!result) {
```

Comment:
! enough here? we store direct values and not "Only objects" as in the cases till now?